kv/collection.go

```diff
@@ -403,3 +406,33 @@ func (c Collection[K, V]) UnlessNotEmpty(
 ) Collection[K, V] {
 	return c.WhenEmpty(f)
 }
+
+func (c Collection[K, V]) Random(v ...int) Collection[int, V] {
+	n := 1
+	if len(v) > 0 {
+		n = v[0]
+	}
+
+	values, _ := c.RandomE(n)
+
+	return CollectSlice(values)
+}
+
+func (c Collection[K, V]) RandomE(n int) ([]V, error) {
+	cLen := len(c.values)
+
+	if n > cLen {
+		return nil, errors.NewInvalidArgumentError(
+			fmt.Errorf("you requested %d items, but there are only %d items available", n, cLen),
+		)
+	}
```

Comment:
What do you think about creating a test scenario for this? I think it's the reason that the code coverage stage is returning a lower coverage